src/organizercore.cpp

```diff
@@ -1599,43 +1592,3 @@ void OrganizerCore::prepareStart() {
   storeSettings();
 }
 
-/*

```

Comment:
This is an unnecessary diff. This code is actually used (in modified form) in the branch so this diff will cause an avoidable conflict.